src/runtime/node/buffer/$cloudflare.ts

```diff
@@ -1,9 +1,9 @@
 // https://nodejs.org/api/buffer.html
 import type nodeBuffer from "node:buffer";
 
-export { INSPECT_MAX_BYTES, resolveObjectURL } from "./index";
+export { atob, btoa, INSPECT_MAX_BYTES, resolveObjectURL } from "./index";
 
-import { INSPECT_MAX_BYTES, resolveObjectURL } from "./index";
+import { atob, btoa, INSPECT_MAX_BYTES, resolveObjectURL } from "./index";
```

Comment:
I think this is the fix (adding missing exports), we should have reflected it in the title.